mrc-ide/hint
# New ADR resources should grant permissions to UNAIDS org

## Description

To test:
- If they exist delete Naomi zip and report from [Antarctica HIV Estimates 2022](https://dev.adr.fjelltopp.org/country-estimates-22/antarctica-country-estimates-2022-1)
- Make the dataset public:
    ```shell
    curl -s "https://dev.adr.fjelltopp.org/api/3/action/package_patch" -F id=antarctica-country-estimates-2022-1 -H "Authorization: ${ADR_API_KEY?}" -F private=false
    ```
- Upload Naomi zip using `master` branch
- Upload Naomi report using this branch
- Go to ADR and compare access restrictions applied to zip and report on their respective details pages. Nothing should be shown for zip. For reports it should say "Access restricted to specific organizations" and hovering will display `unaids` as well as `naomi-development-team`. Tomek has confirmed this is desired behaviour.
![Screenshot from 2021-10-19 15-30-39](https://user-images.githubusercontent.com/1724545/137931743-69e4ab1f-acc0-460f-b6c2-34f641570ec5.png)


## Type of version change

Minor

## Checklist
